UnityProject/Assets/Scripts/Managers/NetworkManagement/CustomNetworkManager.cs

```diff
@@ -77,9 +84,40 @@ public void UpdateMe()
 	}
 
 
+	public void ReceiveMattOverrides(MapSaver.MapSaver.CompactObjectMapData data, bool DoStraightaway)
```

Comment:
```cs
public void ReceiveMattOverrides(MapSaver.MapSaver.CompactObjectMapData data, bool DoStraightaway)
{
    foreach (var PD in data.PrefabData)
    {
        var netID = data.IDToNetIDClient[PD.GitID];
        PrePayload[netID] = PD;

        if (DoStraightaway && Spawned.TryGetValue(netID, out var networkIdentity))
        {
            ObjectBeforePayloadDataClient(networkIdentity);
        }
    }
}

```

Reply:
nah, you need all references because it might point to an object that is in the queue already